docs/section21/Rule21-3.md

```diff
@@ -0,0 +1,59 @@
+
+# Boiler - Rule 21-3  
+
+**Rule ID:** 21-3  
+**Rule Description:** Heating hot water plant capacity shall be based on coincident loads.  
+**Rule Assertion:** B-RMR FluidLoop.heating_design_and_control: is_sized_using_coincident_load = True  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.2.2 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B-RMR  
+**Applicability Checks:**  
+
+1. P-RMR is modeled with purchased heating.
+2. B-RMR is modeled with at least one air-side system that is Type-1a, 7, 11, 12 or that is Type-1, 5, 7, 11, 12.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:**  
+
+1. check_purchased_chw_hhw()
+
+**Applicability Check:**
+
+1. Check if P-RMR is modeled with purchased cooling or purchased hot water/steam: `purchased_chw_hhw_status_dict = check_purchased_chw_hhw(P_RMR)`
+
+  - If P-RMR is modeled with purchased hot water/steam, rule is not applicable: `if purchased_chw_hhw_status_dict["PURCHASED_HEATING"]: rule_applicability_flag = FALSE`
+
+  - Else, P-RMR is not modeled with purchased hot water/steam, continue to next applicability check: `if NOT purchased_chw_hhw_status_dict["PURCHASED_HEATING"]:`
+
+2. B-RMR is modeled with at least one air-side system that is Type-1a, 7, 11, 12 or that is Type-1, 5, 7, 11, 12:
+
```

Comment:
@xing not sure I understand why we have listed the systems twice. Is one set for the modified baseline purchased H system? If yes, maybe we can come up with a different terminology for them maybe 'G3.1.1.3.3. baseline systems for PCHW and PHW'

Reply:
I agree. I would just use one list here 1a, a, 5,7,11,12